package.json

```diff
@@ -98,7 +98,7 @@
     "@hot-loader/react-dom": "^16.13",
```

Comment:
Text not inverted to black on Light Theme, I think

![image](https://user-images.githubusercontent.com/64950861/147685247-56539e5b-ab25-4004-bca8-1bc9644eb7ea.png)

Reply:
Fixed.
![image](https://user-images.githubusercontent.com/94839278/147685624-224f0f66-d060-494c-a417-9deee931ddcf.png)